### DISCUSSION TITLE
LoginAs not a function

### DISCUSSION THREAD

**Discussion by victorLessa** [adonisjs/core#4298] [Help]
My application is not recognizing the loginAs function in tests. Can someone help me?
![image](https://github.com/adonisjs/core/assets/33801047/4d63236a-3813-47ee-9085-d6a4621806ac)
![image](https://github.com/adonisjs/core/assets/33801047/c98eab35-9597-466e-b4db-e175499d7108)

authentication works when tested manually using postman

**Comment by emilenasc** [comment]
export const plugins: Config['plugins'] = [assert(), apiClient(), pluginAdonisJS(app), authApiClient(app)] 

hi, :) you have to add authApiClient(app) in your plugins in tests/bootstrap.ts
